front/src/modules/ui/layout/styles/texts.ts

```diff
@@ -0,0 +1,16 @@
+export const commonText = {
+  fontSizeExtraSmall: '0.85rem',
+  fontSizeSmall: '0.92rem',
+  fontSizeMedium: '1rem',
+  fontSizeLarge: '1.08rem',
+
+  iconSizeSmall: '0.92rem',
+  iconSizeMedium: '1.08rem',
+  iconSizeLarge: '1.23rem',
+
+  fontWeightMedium: 500,
+  fontWeightSemibold: 600,
+  fontWeightBold: 700,
+  fontFamily: 'Inter, sans-serif',
+  lineHeight: '150%',
```

Comment:
not related to your PR, but this feels wrong to me and should not belong to the theme: 150% depends on the container